### ISSUE TITLE
Latex equation formula doesn't work

### ISSUE THREAD

**Issue by agrgal** [xournalpp/xournalpp#5215]
### Operating System

Linux

### (Linux only) Distribution

Ubunt 22.04.3 LTS

### (Linux only) Desktop Environment

Mate 1.26.0

### (Linux Only) Display Server

X.Org

### Installation Method

deb package

### Xournal++ Version

1.2.1

### libgtk Version

3.24.33

### Bug Description

I can't use the latex equation button.

### Expected Behaviour

Use it, of course. :-)

### Steps to Reproduce

I tried to insert a TEX formula. First, I get default_template.tex doesn't found warning. I created in my home xournal directory (/home/user/.config/xournal) a basic template file (but don't know if suitable for this specific purpose)

> \documentclass[11pt]{article}
> \begin{document}
> \end{document}

Later, I got the issue. 
![error](https://github.com/xournalpp/xournalpp/assets/12474907/1080e1d7-3a49-4318-9d38-83107987d2df)

**Comment by rolandlo** [comment]
No, the template you defined is not usable. The `default_template.tex` is in `/usr/share/xournalpp/resources/default_template.tex`. You could also download it from the [repo](https://raw.githubusercontent.com/rolandlo/xournalpp/master/resources/default_template.tex) and put it wherever you want.

**Comment by rolandlo** [comment]
By the way `/home/user/.config/xournal` has nothing to do with `xournalpp`.

**Comment by mennucc** [comment]
if you have installed using the `snap` repository, then the template is in `/snap/xournalpp/current/usr/share/xournalpp/resources/default_template.tex`

**Comment by agrgal** [comment]
Thanks both of u. Yes, I checked it, and default_template.tex is in the correct path as @rolandlo commented. But the issue **filesystem error** still stands. When I close that popup window, try to insert a latex equation and crashes when accepted.

**Comment by rolandlo** [comment]
It looks that no PDF is generated.

- What is your LaTeX generation command? Can you post a screenshot of your settings in the LaTeX panel of menu `Edit > Preferences`?
- Is `pdflatex` (or the generator you use) in PATH?
- Can you upload the file `/tmp/xournalpp-37601/tex/tex.tex` and also check if it compiles correctly with `pdflatex` (or the generator you use)? Also do you get any message in the command output tab in the LaTeX tool editor? Can you upload the corresponding `tex.log` file if existing. You will need to replace the number (37601) if you give it a fresh try.
